### PR TITLE
C388515 Click on "Link headings" button when edit "MARC bib" >> unlink some fields >> click on "Link headings" button again (spitfire) (TaaS)

### PR DESCRIPTION
**Pull request by elen2345**
[TestRail - 388515](https://foliotest.testrail.io/index.php?/cases/view/388515)
[Allure Report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/6016/allure/)
image

### CODE REVIEW

File: cypress/e2e/marc/marc-bibliographic/edit-marc-bib/automated-linking/edit-marc-bib-link-headings-unlink-link-again.cy.js [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+        linkedFields[0].rowIndex,
+        linkedFields[0].tag,
+        '1',
+        '\\',
+        '$a Lee, Stan, $d 1922-2018',
+        '$e author.',
+        '$0 id.loc.gov/authorities/names/n83169267',
+        '',
+      );
+      // #6 Edit subfield "$0" value of unlinked field to another valid (matched with "naturalId" of existing "MARC authority" record)
```
Comment (manowarqqq) [inline cypress/e2e/marc/marc-bibliographic/edit-marc-bib/automated-linking/edit-marc-bib-link-headings-unlink-link-again.cy.js:198]: Missed expected result for Step6, Step7

File: cypress/e2e/marc/marc-bibliographic/edit-marc-bib/automated-linking/edit-marc-bib-link-headings-unlink-link-again.cy.js [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+      QuickMarcEditor.checkCallout(
+        'Field 600 and 650 has been linked to MARC authority record(s).',
+      );
+      QuickMarcEditor.verifyRowLinked(linkedFields[0].rowIndex, true);
+      QuickMarcEditor.verifyRowLinked(linkedFields[3].rowIndex, true);
+      QuickMarcEditor.verifyDisabledLinkHeadingsButton();
+      // #9 Click on the "Save & keep editing" button.
+      QuickMarcEditor.clickSaveAndKeepEditing();
+      linkedFields.forEach((field, index) => {
+        if (index !== 4) {
```
Comment (manowarqqq) [inline cypress/e2e/marc/marc-bibliographic/edit-marc-bib/automated-linking/edit-marc-bib-link-headings-unlink-link-again.cy.js:219]: not clear what we verify here. Magic numbers

File: cypress/e2e/marc/marc-bibliographic/edit-marc-bib/automated-linking/edit-marc-bib-link-headings-unlink-link-again.cy.js (hunk 1 above)
Comment (elen2345) [reply cypress/e2e/marc/marc-bibliographic/edit-marc-bib/automated-linking/edit-marc-bib-link-headings-unlink-link-again.cy.js:213]: Done

File: cypress/e2e/marc/marc-bibliographic/edit-marc-bib/automated-linking/edit-marc-bib-link-headings-unlink-link-again.cy.js (hunk 2 above)
Comment (elen2345) [reply cypress/e2e/marc/marc-bibliographic/edit-marc-bib/automated-linking/edit-marc-bib-link-headings-unlink-link-again.cy.js]: Fixed

**Review by manowarqqq — APPROVED** [review]

**Review by ostapwd — APPROVED** [review]

**Pull request merged**